src/components/ActionMenuHelper/styles.styl

```diff
@@ -0,0 +1,3 @@
+.ActionMenuHelper__menu
+    position relative
+    top  .5rem
```

Comment:
Is it the behavior we want for all our ActionMenu ?

Reply:
Yep I think so. @joel-costa, the margin between the button and the menu here is expected to be in all actions menu ?

<img width="283" alt="image" src="https://user-images.githubusercontent.com/465582/82217505-cf0a9c80-991a-11ea-8d62-745ab5ba2d05.png">

@Crash-- there is already a discussion on cozy-ui on ActionMenuHelper here : https://github.com/cozy/cozy-ui/issues/1463. I plan to move the component ASAP.

Reply:
Thanks for pointing the issue!